alerter-component/src/main/java/io/github/steliospaps/ighackathon/instrumentpricealerter/alertercomponent/websocket/client/WebsocketClient.java

```diff
@@ -258,4 +265,10 @@ public Health health() {
 		}
 	}
 
+	@Scheduled(fixedRate = 10_000, initialDelay = 15_000)
+	public void resetWsMessageReceivedCount() {
```

Comment:
are we doing this because there is no good way to plot counters in aws?

Reply:
The micrometer counter monotonically increases and I couldnt see a way of using it to log the delta within a 30 sec time period. So I used an additional variable for logging purposes. Cloudwatch graphs appear to be doing the server side rate aggregation function.